topics/nodejs/index.md

```diff
@@ -11,4 +11,4 @@ topic: nodejs
 url: https://nodejs.org/en/
 wikipedia_url: https://en.wikipedia.org/wiki/Node.js
 ---
-Node.js was written by Ryan Dahl in May of 2009. It enables JavaScript to run on the server, rather than the client, side, which allows for faster response time when users interact with a webpage.
+Node.js was written by Ryan Dahl in May of 2009. It enables JavaScript to run on the server, rather than the client-side, which allows for faster response time when users interact with a webpage.
```

Comment:
I think this was intentional; @nayafia to confirm. 🤔  We could rephrase to "on the server-side, rather than the client-side, which allows" for clarity.

Reply:
ah yes. I think it was intentional too. I think `on the server side, rather than the client side` (no hyphens) would clarify, if you want to update it @morkro!